drivers/overlay/encryption.go

```diff
@@ -31,6 +31,12 @@ const (
 	bidir
 )
 
+var (
+	saTxHardLimit = netlink.XfrmStateLimits{TimeHard: uint64(12*3600 + 60)}
+	saRxHardLimit = netlink.XfrmStateLimits{TimeHard: uint64(3*12*3600 + 60)}
```

Comment:
These are hard limits, so independent of traffic hit.
Yes, they will kick in if the daemon is down or if no more encrypted networks are present.

If an encrypted network is present, the hard timeout will never kick in, because we replace the entries with new ones (at key rotation) before the timers expire.